.teamcity/_self/projects/WPComTests.kt

```diff
@@ -207,6 +207,178 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
+            logs.tgz => logs.tgz
+            screenshots => screenshots
+        """.trimIndent()
+
+        vcs {
+            root(Settings.WpCalypso)
+            cleanCheckout = true
+        }
+
+        params {
+            text(
+                name = "URL",
+                value = "https://wordpress.com",
+                label = "Test URL",
+                description = "URL to test against",
+                allowEmpty = false
+            )
+            checkbox(
+                name = "GUTENBERG_EDGE",
+                value = "false",
+                label = "Use gutenberg-edge",
+                description = "Use a blog with gutenberg-edge sticker",
+                checked = "true",
+                unchecked = "false"
+            )
+            checkbox(
+                name = "COBLOCKS_EDGE",
+                value = "false",
+                label = "Use coblocks-edge",
+                description = "Use a blog with coblocks-edge sticker",
+                checked = "true",
+                unchecked = "false"
+            )
+        }
+
+        steps {
+            bashNodeScript {
+                name = "Prepare environment"
+                scriptContent = """
+                    export NODE_ENV="test"
+                    export PLAYWRIGHT_BROWSERS_PATH=0
+
+                    # Install modules
+                    ${_self.yarn_install_cmd}
+
+                    # Build packages
+                    yarn workspace @automattic/calypso-e2e build
+                """
+            }
+            bashNodeScript {
+                name = "Run e2e tests ($viewportName)"
+                scriptContent = """
+                    shopt -s globstar
+                    set -x
+
+                    chmod +x ./bin/get-calypso-live-url.sh
+                    URL=${'$'}(./bin/get-calypso-live-url.sh ${BuildDockerImage.depParamRefs.buildNumber})
+                    if [[ ${'$'}? -ne 0 ]]; then
+                        // Command failed. URL contains stderr
+                        echo ${'$'}URL
+                        exit 1
+                    fi
+
+                    cd test/e2e
+                    mkdir temp
+
+                    export LIVEBRANCHES=true
+                    export NODE_CONFIG_ENV=test
+                    export PLAYWRIGHT_BROWSERS_PATH=0
+                    export TEAMCITY_VERSION=2021
+
+                    # Decrypt config
+                    openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%CONFIG_E2E_ENCRYPTION_KEY%"
+
+                    # Run the test
+                    export VIEWPORT_NAME=$viewportName
+                    export LOCALE=en
+                    export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
+                    export DEBUG=pw:api
+
+                    xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --testNamePattern @parallel --maxWorkers=%E2E_WORKERS% specs/specs-playwright-wpcom
```

Comment:
This is just a placeholder while we figure out how best to organize the different spec files.
More details in pciE2j-sq-p2